packages/logs/src/boot/startLogs.spec.ts

```diff
@@ -161,6 +163,10 @@ describe('logs', () => {
     })
 
     it('should send bridge event when bridge is present', () => {
+      if (!isChromium()) {
+        pending('no ReportingObserver support')
+      }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏how is this test related to `ReportingObserver`?

Reply:
Bad copy past..